packages/kbn-storybook/storybook_config/config.js

```diff
@@ -27,7 +27,7 @@ import { create } from '@storybook/theming';
 if (process.env.NODE_ENV === 'test') {
   // eslint-disable-next-line
   require('babel-plugin-require-context-hook/register')();
-} else {
+} else if (!process.env.PERCY_TOKEN) {
```

Comment:
I don't love this either. We're deciding not to add all the storybook decorations if we have a PERCY_TOKEN defined in the environment. I couldn't find a great way to pass in flags to a storybook config but I'm def open to other suggestions